### PR TITLE
Items Update

### PR DESCRIPTION
**Pull request by BrOgBr**
* **Addressed Issue(s)**: -

* **Server Mode**:  Renewal

* **Description of Pull Request**: 

* Fixed some locations and scripts;
* Added new item;
* Added and fixed combos.

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -10797,7 +10797,6 @@
 20346,C_Drooping_Permeter,Costume Drooping Permeter,4,0,,0,,,,0,0xFFFFFFFF,63,2,1024,,1,1,694,{},{},{}
 20347,C_Poring_Pirate_Hat,Costume Poring Pirate Hat,4,0,,0,,,,0,0xFFFFFFFF,63,2,1024,,1,1,726,{},{},{}
 20348,C_Sunday_Hat,Costume Sunday Hat,4,0,,0,,,,0,0xFFFFFFFF,63,2,1024,,1,1,117,{},{},{}
-20349,C_Flying_Galapago,Costume Flying Galapago,4,0,,0,,,,0,0xFFFFFFFF,63,2,4096,,1,0,1358,{},{},{}
```
Comment (Lemongrass3110) [inline db/re/item_db.txt]: Why remove this?

File: db/re/item_combo_db.txt [hunk 2]
```diff
@@ -603,7 +603,7 @@
 18985:28321:28322,{ bonus2 bSkillAtk,"HT_BLITZBEAT",200; }
 18985:28322,{ bonus2 bSkillAtk,"SN_FALCONASSAULT",50; }
 18997:28326:28327,{ bonus bSpeedRate,10; bonus bBaseAtk,50; bonus bMatk,50; if(getequiprefinerycnt(EQI_HEAD_TOP)>=9){ bonus bMaxHPrate,10; bonus bMaxSPrate,50; } }
-19026:20756,{ .@r = getequiprefinerycnt(EQI_GARMENT); bonus2 bSPGainRace,RC_Fish,10; bonus3 bAddMonsterDropItem,551,RC_Fish,(5+.@r); bonus2 bAddItemHealRate,551,5; bonus bHPrecovRate,(5+.@r); }
+19026:20756,{ .@r = getequiprefinerycnt(EQI_GARMENT); bonus2 bSPGainRace,RC_Fish,10; bonus3 bAddMonsterDropItem,544,RC_Fish,(5+.@r); bonus3 bAddMonsterDropItem,551,RC_Fish,(5+.@r); bonus2 bAddItemHealRate,544,5; bonus2 bAddItemHealRate,551,5; bonus bHPrecovRate,(5+.@r); }
```
Comment (Lemongrass3110) [inline db/re/item_combo_db.txt]: I dont see general HP recovery in the item descriptions?

File: db/re/item_db.txt (hunk 1 above)
Comment (BrOgBr) [reply db/re/item_db.txt]: Mistake probably, sorry.

File: db/re/item_combo_db.txt (hunk 2 above)
Comment (BrOgBr) [reply db/re/item_combo_db.txt]: You're right Lemon, I forgot to remove it.

**Comment by Atemo** [comment]
Anything left ? I prefer to review in one go.

**Comment by BrOgBr** [comment]
> Anything left ? I prefer to review in one go.

Not for now, without time to do more.

File: db/re/item_combo_db.txt [hunk 3]
```diff
@@ -603,7 +603,7 @@
 18985:28321:28322,{ bonus2 bSkillAtk,"HT_BLITZBEAT",200; }
 18985:28322,{ bonus2 bSkillAtk,"SN_FALCONASSAULT",50; }
 18997:28326:28327,{ bonus bSpeedRate,10; bonus bBaseAtk,50; bonus bMatk,50; if(getequiprefinerycnt(EQI_HEAD_TOP)>=9){ bonus bMaxHPrate,10; bonus bMaxSPrate,50; } }
-19026:20756,{ .@r = getequiprefinerycnt(EQI_GARMENT); bonus2 bSPGainRace,RC_Fish,10; bonus3 bAddMonsterDropItem,551,RC_Fish,(5+.@r); bonus2 bAddItemHealRate,551,5; bonus bHPrecovRate,(5+.@r); }
+19026:20756,{ .@r = getequiprefinerycnt(EQI_GARMENT); bonus2 bSPGainRace,RC_Fish,10; bonus3 bAddMonsterDropItem,544,RC_Fish,(5+.@r); bonus3 bAddMonsterDropItem,551,RC_Fish,(5+.@r); bonus2 bAddItemHealRate,544,5; bonus2 bAddItemHealRate,551,5; }
```
Comment (Atemo) [inline db/re/item_combo_db.txt]: The recovery rate of the items is 5% + 5% per garment refine rate, it should be
`bonus2 bAddItemHealRate,544,5*(1+.@r); bonus2 bAddItemHealRate,551,5*(1+.@r);`

File: db/re/item_combo_db.txt [hunk 4]
```diff
@@ -661,6 +661,7 @@
 20718:22116,{ bonus bMaxHPRate,15; bonus bMaxSPRate,5; }
 20718:22117,{ bonus bMaxHPRate,15; bonus bMaxSPRate,5; }
 20718:22118,{ bonus bMaxHPRate,15; bonus bMaxSPRate,5; }
+20783:4174,{ bonus2 bSubEle,Ele_Neutral,-50; }
 20797:2339,{ bonus bInt,15; bonus bMaxHPrate,5; bonus bMaxSPrate,5; if (getequiprefinerycnt(EQI_ARMOR) > 6) bonus bNoCastCancel; }
 20797:4174,{ bonus2 bSubEle,Ele_All,50; bonus2 bSubEle,Ele_Neutral,-50; }
```
Comment (Atemo) [inline db/re/item_combo_db.txt]: While I see it, the combo should be `20797:4174,{ bonus2 bSubEle,Ele_All,50; }`
Can you change it too?

File: db/re/item_combo_db.txt [hunk 5]
```diff
@@ -805,6 +806,8 @@
 24249:24254,{ bonus2 bMagicAddRace,RC_All,1; bonus bMaxSPrate,1; if (getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L)>=10) { bonus2 bMagicAddRace,RC_All,2; bonus bMaxSPrate,2; } }
 24253:28391,{ bonus2 bAddRace,RC_All,1; bonus bMaxHPrate,1; if (getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L)>=10) { bonus2 bAddRace,RC_All,2; bonus bMaxHPrate,2; } }
 24255:28392,{ bonus2 bAddRace,RC_All,1; bonus bMaxHPrate,1; if (getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L)>=10) { bonus2 bAddRace,RC_All,2; bonus bMaxHPrate,2; } }
+20701:2165,{ .@a = getequiprefinerycnt(EQI_GARMENT); .@b = getequiprefinerycnt(EQI_HAND_L); bonus2 bSubEle,Ele_Earth,.@b; bonus2 bSubRace,RC_Fish,.@a; }
```
Comment (Atemo) [inline db/re/item_combo_db.txt]: The combo increase the damage to earth monster by 1% per refine
`bonus2 bSubEle,Ele_Earth,.@b;` should be `bonus2 bAddEle,Ele_Earth,.@b;`

File: db/re/item_combo_db.txt (hunk 4 above)
Comment (BrOgBr) [reply db/re/item_combo_db.txt]: @Atemo Neutral too?
So 100% def against Neutral element with the Deviling combo.
For me it should be except Neutral.

`20797:4174,{ bonus2 bSubEle,Ele_Water,50; bonus2 bSubEle,Ele_Earth,50; bonus2 bSubEle,Ele_Fire,50; bonus2 bSubEle,Ele_Wind,50; bonus2 bSubEle,Ele_Poison,50; bonus2 bSubEle,Ele_Holy,50; bonus2 bSubEle,Ele_Dark,50; bonus2 bSubEle,Ele_Ghost,50; bonus2 bSubEle,Ele_Undead,50;  }`

Or not?

Comment (Atemo) [reply db/re/item_combo_db.txt]: You're right! Forget it

Comment (BrOgBr) [reply db/re/item_combo_db.txt]: So it's right as I did right?

`20797:4174,{ bonus2 bSubEle,Ele_All,50; bonus2 bSubEle,Ele_Neutral,-50; }`

File: db/re/item_combo_db.txt (hunk 5 above)
Comment (Atemo) [reply db/re/item_combo_db.txt]: For me IRO description match KRO description and the bonus should be
`bonus2 bAddEle,Ele_Earth,.@b; bonus2 bSubRace,RC_Fish,.@a;`

Comment (BrOgBr) [reply db/re/item_combo_db.txt]: Ok, I'll do it. :)

**Pull request merged**